history/history-network.md

```diff
@@ -157,7 +160,7 @@ each receipt/transaction and re-rlp-encode it, but only if it is a legacy transa
 
 HistoricalHashesAccumulatorProof = Vector[Bytes32, 15]
 
-BlockHeaderProof = Union[None, HistoricalHashesAccumulatorProof]
+BlockHeaderProof = Union[HistoricalHashesAccumulatorProof]
```

Comment:
We could for now also leave this `None` in there, allowing this PR to get merged, before clients get ready to migrate data.

And that would allow us to decide on how to deal with it best (future wise).